test/test-api-getreport.js

```diff
@@ -1,11 +1,11 @@
 'use strict';
 
-// Testcase for returning NodeReport as a string via API call
+// Testcase for returning report as a string via API call
 
 const common = require('./common.js');
 const tap = require('tap');
 const nodereport = require('../');
 var report_str = nodereport.getReport();
 common.validateContent(report_str, tap, {pid: process.pid,
-  commandline: [process.argv0, 'test/test-api-getreport.js'].join(' ')
-});
\ No newline at end of file
+  commandline: [process.argv[0], 'test/test-api-getreport.js'].join(' ')
```

Comment:
ah it's because the test script in package.json uses forward slashes so the test does match:
```
"test": "tap test/test*.js"
```

I was using backslashes on the command line when running the individual test locally.